examples/wearable/UIComponents/contents/presentation-views/list/list-button/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		List
+	</title>
+	<link href="../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../css/style.circle.css" rel="stylesheet" />
+	<script src="../../../../js/circle-helper.js"></script>
+	<script src="../../../../lib/tau/wearable/js/tau.js"></script>
+</head>
+
+<body>
+	<div class="ui-page" data-go-to-top-button="true">
+		<header>
+			<h2 class="ui-title">
+				List Button
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a href="three-buttons.html">
```

Comment:
I think more natural is to have Two buttons example first in index

Reply:
List button examples in guide are presented in such order (see both guides html and pdf). How about leving this as it is?